tests/test_code_report.py

```diff
@@ -89,13 +93,15 @@ def test_code_report_extended_arg_search(tmpdir, stdout_checker):
 
     tmpdir.join("source_file.py").write(source_code + '\n')
 
+    cmd = "cd \"{0}\" && " + python() + \
```

Comment:
What is the benefit of introducing this variable?

Reply:
Increased readability. Because of actual variables, inserted variables and a fake variable for a file this config turned into an unreadable mess.

Reply:
Please switch to f-string in both places